internal/learn/networks/iota-1.5X-stardust.md

```diff
@@ -0,0 +1,39 @@
+---
+title: IOTA 1.5x Stardust (Shimmer)
+description: Shimmer is the current staging network of IOTA. New protocol versions are validated in Shimmer before arriving on IOTA Mainnet.
+keywords:
+- incentivized staging network
+- shimmer
+- stardust
+- SMR token
+- migration
+- chrysalis
+---
+## Vision
+
+Stardust is a comprehensive utility upgrade of the IOTA protocol. The main components of Stardust are support for IOTA Smart Contract chains, a new tokenization framework for native assets, revised Dust Protection, and many minor convenience and security updates. With Stardust, IOTA transforms into a multi-asset ledger and enables running smart contracts.
```

Comment:
```suggestion
Stardust is a comprehensive utility upgrade of the IOTA protocol. The main components of Stardust are support for IOTA smart contract (ISC) chains, a new tokenization framework for native assets, revised dust protection, and many minor convenience and security updates. With Stardust, IOTA transforms into a multi-asset ledger and enables running smart contracts.
```

I'm not sure anymore what we agreed on in terms of capitalization. IIRC we capitalize names like IOTA and Stardust, but objects like smart contracts would remain lower case. Or does this count as part of the name of the protocol?

Also since this is the first mention, added the abbreviation.